_inc/client/at-a-glance/scan.jsx

```diff
@@ -61,7 +61,7 @@ const DashScan = React.createClass( {
 							module="scan"
 							status="is-error"
 							statusText={ __( 'Threats found' ) }
-							pro={ true } >
```

Comment:
It's the least obtrusive way to avoid the are toggle being overwritten by https://github.com/Automattic/jetpack/blob/a38c6aa247a0b5fe638811c6bca942e5d3c87f6e/_inc/client/components/dash-item/index.jsx#L106-L117